docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+    
+    dalalserver:
+        container_name: dalalstreet-server
+        build: .
+        volumes: 
+            - ./dalalstreet_docker.log:/dalal-street-server/dalalstreet_docker.log
+        ports:
+            - "${SERVER_PORT}:8000"
+        depends_on:
+            - db
+        env_file:
+            - .env
+        environment:
+            MYSQL_DATABASE: ${DB_NAME}
+            DALAL_ENV: Docker
+            MYSQL_ROOT_PASSWORD: ${DB_PASS}
+            MYSQL_PORT: ${MYSQL_PORT}
+        restart: always
```

Comment:
put this in top after image and container name right? So everything looks consistent